about_us.html

```diff
@@ -501,21 +498,80 @@
         display: block;
     }
 
-    #carousel {
-        top: 60px;
-    }
+    @media only screen and (max-height: 1050px){
+        /*TODO: Hero breaks when the verticle resoltion goes below 950px.
+           Need to set any variables that are effected by this to a static number */
 
-    .arrow {
-        top: 200px;
-    }
+        .hero,
+        .space-bg,
+        .space-bg-overlay {
+            height: calc(72.222vh + 291.666px);
+            z-index: 0;
+        }
+
+        #carousel {
+            /*top: 160px at 1050vh*/
+            /*top: calc(23.3333vh - 85px);*/
+            top: calc(21.111vh + -61.667px);
```

Comment:
I wrote a script to make some of the css calculations for me. So that's just output from that script. I figured it probably doesn't make a difference for the computer, but if its more readable I can change it.